### ISSUE TITLE
Nested Drag and Drop (From parent to child and vice versa)

### ISSUE THREAD

**Issue by guptaashwanee** [atlassian/react-beautiful-dnd#2468]
### Description

As I am working on a project where I want Nested Drag and Drop, in which we can Drag and Drop elements from parent to child and vice versa.

**Comment by NehaElevate** [comment]
Hey,

Have you found regarding this something in react-beautiful-dnd that how we can achieve the above said fucntionality.

**Comment by guptaashwanee** [comment]
No, I am trying to write my own. Do you have any resources or references.

**Comment by NehaElevate** [comment]
I found one library which support nested drag - drop functionality i.e: @atlaskit/tree.
